settings.gradle.kts

```diff
@@ -1,4 +1,12 @@
 pluginManagement {
+	resolutionStrategy {
+		eachPlugin {
+			if (requested.id.id == "biz.aQute.bnd.builder") {
+				useModule("biz.aQute.bnd:biz.aQute.bnd.gradle:4.3.0")
+			}
+		}
+	}
+
```

Comment:
Please add the version to the `plugins` block instead and delete `resolutionStrategy` again.

Reply:
Due to an issue with the gradle plugin portal reported here https://github.com/gradle/plugin-portal-requests/issues/9 version 4.3.0 is not accessible and so we have to use the approach shown until the issue can be resolved. Sorry about that.

Reply:
I see. Thanks for the explanation!